cpp/src/arrow/python/serialize.cc

```diff
@@ -663,6 +663,9 @@ Status CountSparseTensors(
       case SparseTensorFormat::CSR:
         ++num_csr;
         break;
+      case SparseTensorFormat::CSC:
+        // TODO(mrkn): support csc
```

Comment:
Can you open a JIRA for this, if not already done?

Reply:
Oh, I forgot to do it.
Filed https://issues.apache.org/jira/browse/ARROW-7419.